tools/cldr-code/src/main/java/org/unicode/cldr/test/CheckAnnotations.java

```diff
@@ -1,27 +1,73 @@
 package org.unicode.cldr.test;
 
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
 import java.util.List;
+import java.util.Map;
+import java.util.Set;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 import org.unicode.cldr.test.CheckCLDR.CheckStatus.Subtype;
+import org.unicode.cldr.util.CLDRFile;
+import org.unicode.cldr.util.LocaleIDParser;
 import org.unicode.cldr.util.PatternCache;
+import org.unicode.cldr.util.XPathParts;
 
 public class CheckAnnotations extends CheckCLDR {
     private static final Pattern ANNOTATION_PATH = Pattern.compile("//ldml/annotations/.*");
 
+    // Skip tts test for these locale/cp combinations unti
+    // https://unicode-org.atlassian.net/browse/CLDR-18329 is fixed
```

Comment:
I did not use logKnownIssue, as far as I know we cannot do that in consoleCheck tests, not sure what would happen in Survey Tool.